Bottom: 2ad0498a00b3c5315d1b7c27fbe2038ed033f689
Top:    b775ae6bcbb55abb8e429635aeef6a30b3e8b664
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 21:23:34 +0100

Remove hack in draw.c which broke PS (and probably other) output

The set_layer call hacked into DrawLayerGroup to reset compositing tricked
the PS exporter to re-open and overwrite output in the main layer output
file during the fab drawing output.


---

diff --git a/src/draw.c b/src/draw.c
index 234e2ec..a7a1a74 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -695,9 +695,6 @@ DrawLayerGroup (int group, const BoxType *drawn_area)
         rv = 0;
       if (layernum < max_copper_layer && Layer->On)
         dapi->draw_layer (Layer, drawn_area, NULL);
-
-      if (gui->gui) /* HACK: Subcomposite each layer in a layer group separately */
-        gui->set_layer (0, group, 0);
     }
   if (n_entries > 1)
     rv = 1;
